.travis.yml

```diff
@@ -38,6 +43,7 @@ before_install:
   - nvm install "$TRAVIS_NODE_VERSION"
   - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version "$TRAVIS_YARN_VERSION"
   - export PATH=$HOME/.yarn/bin:$PATH
+  - yarn config set ignore-engines true
```

Comment:
I am curious, what was the reason for this?

Reply:
Good question! I encountered this error here https://github.com/facebook/create-react-app/issues/4367#issuecomment-384667501 due to an error

```
error upath@1.0.4: The engine "node" is incompatible with this module. Expected version ">=4 <=9".
```

Opened https://github.com/scalameta/metadoc/pull/92 to properly document this in the contributing guide.